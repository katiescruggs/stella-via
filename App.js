import React from 'react';
import { StyleSheet, Text, View } from 'react-native';

export default class App extends React.Component {
  render() {
    const hello = '';
    return (
      <View style={styles.container}>
        <Text h1>Stella Via</Text>
        <Text>{hello}</Text>
        <Text>whats up?</Text>
      </View>
    );
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#fff',
    alignItems: 'center',
    justifyContent: 'center',
  },
});
