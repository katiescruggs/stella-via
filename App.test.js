import React from 'react';
import App from './App';

import renderer from 'react-test-renderer';

it('renders without crashing', () => {
  const rendered = renderer.create(<App />).toJSON();
  
  expect(rendered).toBeTruthy();
});

it('matches the snapshot', () => {
  const rendered = renderer.create(<App />).toJSON();

  expect(rendered).toMatchSnapshot();
});

