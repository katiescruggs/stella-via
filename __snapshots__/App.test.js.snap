// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "space-around",
      "width": "100%",
    }
  }
>
  <Image
    source={1}
    style={
      Array [
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        Object {
          "height": undefined,
          "width": "100%",
        },
        undefined,
      ]
    }
  />
  <Image
    source={1}
    style={
      Object {
        "height": 200,
        "marginTop": 70,
        "width": "100%",
      }
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "flexWrap": "wrap",
        "justifyContent": "center",
        "marginBottom": 60,
      }
    }
  >
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      hasTVPreferredFocus={undefined}
      hitSlop={undefined}
      isTVSelectable={true}
      nativeID={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Array [
          Object {
            "backgroundColor": "transparent",
          },
          Object {
            "alignItems": "center",
            "backgroundColor": "rgba(40, 38, 64, 0.7)",
            "borderRadius": 20,
            "elevation": 1,
            "justifyContent": "center",
            "margin": 10,
            "shadowColor": "rgba(0, 0, 0, 0.6)",
            "shadowOffset": Object {
              "height": 10,
              "width": 2,
            },
            "shadowOpacity": 0.8,
            "shadowRadius": 2,
          },
        ]
      }
      testID={undefined}
      tvParallaxProperties={undefined}
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "height": 130,
            "justifyContent": "center",
            "width": 130,
          }
        }
      >
        <Image
          source={1}
          style={undefined}
        />
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#fff",
              "fontSize": 18,
              "fontWeight": "bold",
              "marginBottom": 4,
              "textAlign": "center",
            }
          }
        >
          Tonight
        </Text>
      </View>
    </View>
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      hasTVPreferredFocus={undefined}
      hitSlop={undefined}
      isTVSelectable={true}
      nativeID={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Array [
          Object {
            "backgroundColor": "transparent",
          },
          Object {
            "alignItems": "center",
            "backgroundColor": "rgba(40, 38, 64, 0.7)",
            "borderRadius": 20,
            "elevation": 1,
            "justifyContent": "center",
            "margin": 10,
            "shadowColor": "rgba(0, 0, 0, 0.6)",
            "shadowOffset": Object {
              "height": 10,
              "width": 2,
            },
            "shadowOpacity": 0.8,
            "shadowRadius": 2,
          },
        ]
      }
      testID={undefined}
      tvParallaxProperties={undefined}
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "height": 130,
            "justifyContent": "center",
            "width": 130,
          }
        }
      >
        <Image
          source={1}
          style={undefined}
        />
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#fff",
              "fontSize": 18,
              "fontWeight": "bold",
              "marginBottom": 4,
              "textAlign": "center",
            }
          }
        >
          Star Map
        </Text>
      </View>
    </View>
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      hasTVPreferredFocus={undefined}
      hitSlop={undefined}
      isTVSelectable={true}
      nativeID={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Array [
          Object {
            "backgroundColor": "transparent",
          },
          Object {
            "alignItems": "center",
            "backgroundColor": "rgba(40, 38, 64, 0.7)",
            "borderRadius": 20,
            "elevation": 1,
            "justifyContent": "center",
            "margin": 10,
            "shadowColor": "rgba(0, 0, 0, 0.6)",
            "shadowOffset": Object {
              "height": 10,
              "width": 2,
            },
            "shadowOpacity": 0.8,
            "shadowRadius": 2,
          },
        ]
      }
      testID={undefined}
      tvParallaxProperties={undefined}
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "height": 130,
            "justifyContent": "center",
            "width": 130,
          }
        }
      >
        <Image
          source={1}
          style={undefined}
        />
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#fff",
              "fontSize": 18,
              "fontWeight": "bold",
              "marginBottom": 4,
              "textAlign": "center",
            }
          }
        >
          Daily Image
        </Text>
      </View>
    </View>
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      hasTVPreferredFocus={undefined}
      hitSlop={undefined}
      isTVSelectable={true}
      nativeID={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Array [
          Object {
            "backgroundColor": "transparent",
          },
          Object {
            "alignItems": "center",
            "backgroundColor": "rgba(40, 38, 64, 0.7)",
            "borderRadius": 20,
            "elevation": 1,
            "justifyContent": "center",
            "margin": 10,
            "shadowColor": "rgba(0, 0, 0, 0.6)",
            "shadowOffset": Object {
              "height": 10,
              "width": 2,
            },
            "shadowOpacity": 0.8,
            "shadowRadius": 2,
          },
        ]
      }
      testID={undefined}
      tvParallaxProperties={undefined}
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "height": 130,
            "justifyContent": "center",
            "width": 130,
          }
        }
      >
        <Image
          source={1}
          style={undefined}
        />
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#fff",
              "fontSize": 18,
              "fontWeight": "bold",
              "marginBottom": 4,
              "textAlign": "center",
            }
          }
        >
          Explore
        </Text>
      </View>
    </View>
  </View>
</View>
`;
